Task_DEV_2/Task_DEV_2/Program.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+
+namespace Task_DEV_2
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Enter the size of the array :");
+            int arrayLength = Convert.ToInt32(Console.ReadLine());
+
+            int[] numberArray = new int[arrayLength];
+            int min = 1;
+            int max = 0;
+            int numberMin = 0;
+            int numberMax = 0;
+
+            for (int i = 0; i < arrayLength; i++)
+            {
+                if (arrayLength > 20)
```

Comment:
You need to do this check before start of array initialization.

Reply:
If array length > 20 user would like to see error message why his program was stopped.

Reply:
Wrong place for this check.